src/test/java/core/basesyntax/dao/ActivityDaoDbImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.storage.Storage;
+import java.util.Map;
+import java.util.Set;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ActivityDaoDbImplTest {
+
```

Comment:
```suggestion
```
check yourself and remove redundant empty lines all across your project

Reply:
ok